nvchecker_source/mercurial.py

```diff
@@ -0,0 +1,10 @@
+# MIT licensed
+# Copyright (c) 2020 Felix Yan <felixonmars@archlinux.org>, et al.
+
+async def get_version(name, conf, *, cache, **kwargs):
+  url = conf.get('mercurial') + '/json-tags'
```

Comment:
Since the `mercurial` key is required, we'd better `conf['mercurial']` and get a `KeyError` if not given.